collectors/metadata/schemas/single-module.json

```diff
@@ -577,6 +613,37 @@
     "metrics"
   ],
   "$defs": {
+    "_categories": {
+      "type": "object",
+      "description": "Category object, with an entry per category depth.",
+      "properties": {
+        "path0": {
```

Comment:
What is path0-1?

Reply:
Yes, this comes from me, we didn't discuss it yesterday, let me explain:

We need a way to limit people on assigning just the categories we want, but we want that for every level of the "category path".

So, if the path is `data-collection/web-servers` you want to have different values allowed for first level `data-collection` and different for second level.

the path0-1 etc is kind of the way we do it on Learn, so that's the way I am familiar, you then join all the pathX and you have the exact category the thing should be in

Reply:
Thanks for the explanation, but better to update the description and OP (if more detailed explanation is needed).